requirements.txt

```diff
@@ -2,6 +2,7 @@ enum34; python_version < '3'
 multipledispatch
 numpy>=1.10.0
 pandas>=0.18.1
+pyspark==2.2.1
```

Comment:
This should not be in here, since `pyspark` will be an optional dependency. Only hard dependencies go in `requirements.txt`.